### DISCUSSION TITLE
UI Generated Masks Positions in config gave me a chuckle (Beta 0.14)

### DISCUSSION THREAD

**Discussion by Duskspeed** [blakeblackshear/frigate#11743] [General]
Hey guys, trying out the new beta and finding the new UI generated masks automaticly being added to the config a great QoL addition to frigate, I got a little chuckle out of checking my config after creating a new mask however, I'll preface this by saying it's 100% my fault for the way I space and comment my configs and it works perfectly fine, and also can easily be manually cleaned up so unless it spits it out between other arguments it's probably not a bug but kinda funny non the less.

My config prior to generating a mask in the UI:
![before mask is generated](https://github.com/blakeblackshear/frigate/assets/103481526/226bec71-e186-4b92-a475-6559e93f2b19)
My config after generating a mask in the UI: 
In the example it spits out the UI generated mask sometimes under a comment or space or in between a comment or space.
![after mask is generated](https://github.com/blakeblackshear/frigate/assets/103481526/1298effe-46ca-4c71-8715-f008db0ed26d)

Keep up the great work!

**Comment by hawkeye217** [comment]
This is just the way that the yaml parser we're using deals with comments. It could have been something with the way you had the comments spaced or indented. 

It's perfectly valid yaml in the end :)

**Comment by banthungprong** [comment]
I "solved" this by moving the detect block to the end of a camera block (so mask isn't the last entry anymore).
No my comment line stays exactly above the camera name when adding or removing masks
